xml_inscriptions/transcribed/NY.Brook.BM.G.16.320.xml

```diff
@@ -35,9 +35,9 @@
                      <supportDesc ana="#metal.gold">
                         <support>
 <p>Gold finger ring</p>
-<dimensions type="surface" unit="m">
-                                        <height>0.0</height>
-                                        <width>0.016</width>
+<dimensions type="surface" unit="cm">
+                                        <height/>
+                                        <width>1.6</width><!-- diam? -->
```

Comment:
This is fine for the present; for reference, you could represent that as `<dim type="diameter">`